docs/style-guide.adoc

```diff
@@ -526,3 +523,28 @@ RETURN 1;
 ----
 RETURN 1
 ----
+
+== Recommendations
+
+* When using Cypher language constructs in prose, use a monospaced font and follow the styling rules.
+** When referring to labels and relationship types, include the colon like so: `:Label`, `:REL_TYPE`.
+** When referring to functions, use lower camel case and parentheses like so: `shortestPath()`. Arguments should normally not be included.
```

Comment:
"...use lower camel case and parentheses like so:" -> "...lower camel case and parentheses should be used as follows:"